### PR TITLE
go-gl/glfw v3.3 support

### PR DESCRIPTION
**Pull request by pchampio**
Waiting for https://github.com/go-gl/glfw/pull/256 to be merged.
~~ref: https://github.com/go-flutter-desktop/go-flutter/issues/84~~

### CODE REVIEW

**Comment by GeertJohan** [comment]
I still need to take a look at scaling here. It seems that the changes to pixel ratio don't work well on all systems, and don't conform to the dp ratio that is advised by flutter/android guidelines.

**Comment by pchampio** [comment]
PTAL: this looks good on my setup. (`xrandr --output "$external" --scale 1.6x1.6 --mode 2560x1440`)
This fixes the wrong calculation of the pixelRatio for external monitor (was too small).
This fixes the wrong default value of the pixelRatio outputed by glfw-v3.3 for external monitor (too big).

This fixes is a bit of a work-arround, glfw cannot see the `--scale` argument.

To calculate the pixelRatio both ContentScale and dpi of the display is used. I thinks it's the way to go. The value `--scale` doesn't matter for the pixelRatio IMO (it's only a x11 thing).

**Comment by james-lawrence** [comment]
bump (just as a reminder the other PR is merged)

**Comment by ericdallo** [comment]
Any news here @pchampio?

**Comment by pchampio** [comment]
@GeertJohan can you take a quick look?
`hover run -b '@go-glv3.3'`

**Comment by pchampio** [comment]
@jld3103 can you please take a look at this PR, it's only a version bump, I don't think I need the full review of Geert.

**Comment by jld3103** [comment]
LGTM

**Comment by jld3103** [comment]
(Also it fixes #373 somehow)

**Comment by crash-override404** [comment]
//Btw my screen scale why its more bigger than before after update

**Comment by pchampio** [comment]
> //Btw my screen scale why its more bigger than before after update

Do you mean, your initial windows size is bigger than before?

File: application.go [hunk 1]
```diff
@@ -119,6 +119,7 @@ func (a *Application) Run() error {
 		glfw.WindowHint(glfw.Visible, glfw.False)
 	}
 
+	glfw.WindowHint(glfw.ScaleToMonitor, glfw.True)
 	if a.config.windowAlwaysOnTop {
```
Comment (pchampio) [inline application.go]: It comes from this line

Comment (crash-override404) [reply application.go]: I just fix it by change my window size  in options

**Comment by crash-override404** [comment]
The update is compitable with go flutter screen size but not for flutter size

**Pull request merged**